app/javascript/packages/verify-flow/steps/password-confirm/password-confirm-step.tsx

```diff
@@ -98,7 +98,12 @@ function PasswordConfirmStep({ errors, registerField, onChange, value }: Passwor
           }),
           {
             button: ({ children }) => (
-              <HistoryLink basePath={stepPath} step={FORGOT_PASSWORD_PATH}>
+              <HistoryLink
+                basePath={stepPath}
+                step={FORGOT_PASSWORD_PATH}
+                aria-label={t('idv.forgot_password.link_html').replace(`%{link}`, '')}
+                title={`${t('idv.forgot_password.link_text')}`}
```

Comment:
Is the `title` necessary here, or is the `aria-label` achieving what we want? My understanding is that the `title` attribute should rarely be used.

See: https://silktide.com/blog/i-thought-title-text-improved-accessibility-i-was-wrong/

Reply:
No, it's not necessary. I will remove and find an alternative way to perform the corresponding frontend tests. Thanks for the link that is informative